src/main.c

```diff
@@ -193,7 +193,8 @@ UX_FLOW(
 );
 
 void display_settings() {
-    strlcpy((char *) confirmLabel, (N_storage.dataAllowed ? allowed : not_allowed),32);
+    // Review: cast destination string from volatile char to char is that ok?
```

Comment:
Yes it is fine to cast to `char *` in this case. You wouldn't want to do that in a complex function for a variable which is used a lot, and where the compiler could make optimization assumptions that the variable doesn't change between accesses.

Reply:
make sure to replace `32` with `sizeof(confirmLabel)`.